server/webdriver/shared_tests/browser_test.py

```diff
@@ -117,7 +117,6 @@ def test_stat_var_hierarchy(self):
 
     # Search for "male asian " and select the first result
     search_input.send_keys(SEARCH_INPUT)
-    wait_elem(self.driver, By.ID, 'sv-search-spinner')
```

Comment:
interesting problem to have that our tests are now too fast for the spinner to load.
as a UX test, we are verifying that the spinner loads if data is slow. does this test still pass in the record phase if we skip straight to the results?